### ISSUE TITLE
Paradox Plaza Deals not Updated

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#1127]
Price cuts offered on Paradox Plaza that no longer exist still show up as deals. Here are two examples, where the website claims that there is a large discount, when indeed there is none:
https://isthereanydeal.com/game/crusaderkingsii/info/
https://isthereanydeal.com/game/europauniversalisiv/info/

You can look at any game or DLC developed by Paradox, and sold on their website and many have these incorrect deals.

[Issue created by KeetnaWilson: 2018-07-25]
[Last updated on bitbucket: 2019-02-15]

[Comment created by TamamoNoMae: 2019-02-15]
As of right now, Paradox Plaza doesn't show up as a store at all when trying to search for Crusader Kings II DLC. Yet there's a sale going on for a few days for up to 75% on their website.

https://isthereanydeal.com/search/?q=Crusader+Kings+II%3A+Holy+Fury
https://www.paradoxplaza.com/games/?prefn1=pdx-brand&prefv1=Crusader%20Kings

[Comment created by dineapples: 2018-08-02]
+1 and more details:

Using Crusader Kings II: The Reaper's Due as an example: https://isthereanydeal.com/game/crusaderkingsiireapersdue/info/
The "current price" listed for Paradox is $4.99 (-50%), but the actual price on Paradox's page is currently $9.99 (-0%). The price history correctly displays that Paradox had a sale for the item last month, but has not updated the prices since the sale ended. In fact, previous Paradox sales have correctly started and ended, but the most recent one has not been reset. Other sites continue to have correct and current prices.

The status page says that Paradox Games deals and catalog has been updated ~6 minutes ago for US prices.

[Another game](https://isthereanydeal.com/game/magickaii/info/) sold by Paradox Games which is currently on sale (as of 08/03) is correctly priced on IsThereAnyDeal. 

![firefox_2018-08-02_16-04-00.png](https://bitbucket.org/repo/b6qGGA/images/3016709211-firefox_2018-08-02_16-04-00.png)

tl;dr Paradox Games discounts are correctly updated, but since last month, prices aren't reset after sales end. Other merchant prices for the affected games appear correct.
